air-interpreter/Cargo.toml

```diff
@@ -26,4 +26,4 @@ serde = { version = "=1.0.118", features = [ "derive", "rc" ] }
 serde_json = "1.0"
 
 [features]
-fce = ["interpreter-lib/fce"]
+fce = ["air/fce"]
```

Comment:
fce => marine?

Reply:
I'll do it on the next PR with `sdk` renaming.